File: RB101_RB109_small_problems/easy_3/searching_101.rb
SIZE_FOR_ARRAY = 5

def suffix(num)
  case num
  when 1
    "st"
  when 2
    "nd"
  when 3
    "rd"
  else
    "th"
  end
end

arr = []


for i in 1..SIZE_FOR_ARRAY do
  puts "=> Enter the #{i}#{suffix(i)} number:"
  arr << gets.chomp.to_i
end

puts "=> Enter the last number:"
final_num = gets.chomp.to_i

check = arr.include?(final_num)
appears_or_not = check ? "appears" : "does not appear"

puts "The number #{final_num} #{appears_or_not} in #{arr}."

File: JS210_small_problems/easy_3/searching_101.js
let rlSync = require('readline-sync');

let numbers = [];

let iterations = ['1st', '2nd', '3rd', '4th', '5th', 'last'];

iterations.forEach(function(seq) {
  let num = Number.parseInt(rlSync.question(`Enter the ${seq} number: `), 10);
  numbers.push(num);
});

let lastNum = numbers.pop();

let appearsString = numbers.includes(lastNum) ? 'appears' : 'does not appear';

let output = `The number ${String(lastNum)} ${appearsString} in [${numbers.join(', ')}].`;

console.log(output);

// Further Exploration

function isIncluded(arr, val) {
  return arr.some(element => element > val);
}